README.md

```diff
@@ -1,9 +1,28 @@
 AI for the [2048 game](http://gabrielecirulli.github.io/2048/). This uses *expectimax optimization*, along with a highly-efficient bitboard representation to search upwards of 10 million moves per second on recent hardware. Heuristics used include bonuses for empty squares and bonuses for placing large values near edges and corners. Read more about the algorithm on the [StackOverflow answer](https://stackoverflow.com/a/22498940/1204143).
 
+## Features
```

Comment:
This whole section is pointless and essentially rewords the previous paragraph. It also sounds like it was written rather poorly by an AI (well, your whole PR kind of sounds AI-written, sorry).